connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1654,7 +1748,9 @@ ble_error_t Gap::createAdvertisingSet(
     const AdvertisingParameters &parameters
 )
 {
+    tr_info("Create advertising set");
```

Comment:
Can you detail the advertising parameters ?

Reply:
Yes, but it will be repeated if `setAdvertisingParameters` is called later.